### DISCUSSION TITLE
merge different video formats automatically optimized?

### DISCUSSION THREAD

**Discussion by Clemens-Ratte-Polle** [mifi/lossless-cut#753] [Ideas]
Hi :)
When merging different video formats and sizes you mostly have to re-encode ALL videos. This could be optimized.
Perhaps Lossless Cut could automtically analyze all videos to show some options and suggestions to make an optimized conversion.
No all videos have to be new encoded, so you do it only with the lesser videos of one format to the format of the most video parts, which could be time saving.
But you perhaps want to upscale lower resolutions to the higher video party.
Here Lossless-Cut could give some options automatically :)

**Comment by mifi** [comment]
Hi! Nice idea. But Losslesscut doesn't really do lossy operations, so I don't think this is something that will ever be prioritized, unfortunately.
